### ISSUE TITLE
Layout/ArgumentAlignment: EnforcedStyle: with_fixed_indentation is broken

### ISSUE THREAD

**Issue by dominikduda** [rubocop/rubocop#9944]
## Expected behavior

With the following setting:
```
Layout/ArgumentAlignment:
  EnforcedStyle: with_fixed_indentation
Layout/LineLength:
  Max: 120
```
The following code:

```ruby
  form do |f|
    f.semantic_errors
    f.inputs do
      input :name
      input :organization_users, as: :select, multiple: true, collection: current_organization.organization_users.joins(:user).order('users.last_name')
      input :uuid
    end
    f.actions
  end
```
Should be formatted to the following with autoformat:

```ruby
  form do |f|
    f.semantic_errors
    f.inputs do
      input :name
      input :organization_users, as: :select, multiple: true, 
        collection: current_organization.organization_users.joins(:user).order('users.last_name')
      input :uuid
    end
    f.actions
  end
```

## Actual behavior

But it actually is formatted as follows (0 indent):
```ruby
  form do |f|
    f.semantic_errors
    f.inputs do
      input :name
      input :organization_users, as: :select, multiple: true,
collection: current_organization.organization_users.joins(:user).order('users.last_name')
      input :uuid
    end
    f.actions
  end

```

## Steps to reproduce the problem

Use config from the top of this issue.

Run autoformat against the following file:

```ruby
ActiveAdmin.register ClinicalGroup do
  menu priority: 2

  controller do
    def scoped_collection
      end_of_association_chain.where(organization: current_organization)
    end
  end

  before_create do |group|
    group.organization = current_organization
  end

  permit_params :name, :uuid, organization_user_ids: []

  form do |f|
    f.semantic_errors
    f.inputs do
      input :name
      input :organization_users, as: :select, multiple: true, collection: current_organization.organization_users.joins(:user).order('users.last_name')
      input :uuid
    end
    f.actions
  end

  index do
    column :id
    column :name
    column('Users') { |group| group.users.map(&:full_name).to_sentence }
    column :uuid
    actions
  end

  filter :name_contains

  show do
    attributes_table do
      row :id
      row :name
      row('Users') { |group| group.users.map(&:full_name).to_sentence }
      row :uuid
    end
  end
end
```

## RuboCop version

1.18.3 (using Parser 3.0.2.0, rubocop-ast 1.8.0, running on ruby 3.0.2 x86_64-linux)

Edit:
Tinkering with `IndentationWidth` option does not help.

**Comment by dominikduda** [comment]
Any news about this?

**Comment by koic** [comment]
This issue has been fixed with the latest RuboCop (v1.25.0) . Can you update to the latest version? Thank you.

**Comment by dominikduda** [comment]
Thanks!
